test/evergreen.yml

```diff
@@ -3370,6 +3370,21 @@ tasks:
         vars:
           perf-test-name: modify-force-update-large-record-btree.wtperf
 
+  - name: perf-test-mongodb-oplog
+    tags: ["btree-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: mongodb-oplog.wtperf
+          maxruns: 3
+          wtarg: -ops ['"load", "insert", "truncate"']
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: mongodb-oplog.wtperf
```

Comment:
I think we should only add the tests that are required. @agorrod If these tests are required then I do not mind adding them.

Reply:
Interesting. I hadn't looked at those workloads. There are two `mongodb-small-oplog.wtperf` and `mongodb-large-oplog.wtperf` that run for an hour. Could they be added to the long perf jobs?

I think the `secondary-apply` one is interesting and should be added.

Reply:
In that case @agorrod, shall we just create a new tag `oplog-perf`, and group them all together, as @ajmorton suggested?

Reply:
Sounds good to me.

Reply:
Thanks, @agorrod for some reason I just saw this today, I will get working on this ticket.

Reply:
@agorrod Looking at the [mongodb-oplog.wtperf](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-oplog.wtperf), [mongodb-small-oplog.wtperf](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-small-oplog.wtperf), and [mongodb-large-oplog.wtperf](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-large-oplog.wtperf) config files, they look very much alike. Is it worth it to add all of them in evergreen, given that they take at least 2 hours to run? 

Perhaps we could stick with one of these ([mongodb-oplog.wtperf](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-oplog.wtperf) or [mongodb-small-oplog.wtperf](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-small-oplog.wtperf))?

Also, I do not see any stat info for [secondary-apply](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/runners/mongodb-secondary-apply.wtperf) in `test.stat` that the wtperf outputs. So I don't think it is worth it to add this test either.

Moreover, I am also thinking of adding another stat for the `disk size` for the oplog test, let me know if that sounds like a good thing to do.

Reply:
The tasks use different insert rates and key sizes (I'm not sure why they didn't use data size).
They seem like relevant things to check, MongoDB users do have different size entries, and simulating those feels relevant.

`secondary-apply` outputs inserts when I run it, for example: `80000 inserts`. I think that workload is crafted to be able to maintain that throughput. If it doesn't there is something to follow up on.

Adding a stat for `database size` does feel worthwhile to me.

Reply:
Thanks, @agorrod, that makes sense to me. However, as I have mentioned before, if we run these 4 tasks (3 times each), it will take at least ~8 hours. Shall we reduce that frequency to maybe a 2 or a 1?

Reply:
I like running these tests just once. 
Looking at the patches we're not seeing any heavy variance from test to test and if the goal of these tests is to catch a large change in final DB size that should be visible without averaging multiple runs per commit.